__tests__/integration/leaveEditor.ppt.js

```diff
@@ -25,7 +25,7 @@ describe('Leaving the editor', () => {
     await pupExpect(page).toMatch('Load by URI')
 
     // Navigate back
-    await pupExpect(page).toClick('a.nav-link', { text: 'Editor' })
+    await pupExpect(page).toClick('a.nav-link', { text: /^Editor$/ })
```

Comment:
That's odd.

Reply:
Yeah, had to use a regex because the test was matching the *Profile Editor* instead of just matching the  *Editor* button.